argo/cron/cdc-file-sort.yaml

```diff
@@ -0,0 +1,120 @@
+metadata:
+  name: tipocket-cdc
```

Comment:
We should rename `tipocket-cdc`

Reply:
oh, yes

Reply:
fixed in https://github.com/pingcap/tipocket/pull/298